solutions/instances/variables.tf

```
@@ -321,7 +328,7 @@ variable "existing_cos_kms_key_crn" {
 
 variable "kms_endpoint_type" {
   type        = string
-  description = "The type of endpoint to use to communicate with the key management service (KMS). Specify one of the following values for the endpoint type: `public` or `private` (default)."
+  description = "The type of endpoint to use to communicate with the key management service (KMS). Specify one of the following values for the endpoint type: `public` or `private` (default). To support cross account functionality you must also provide a value for `ibmcloud_kms_api_key`."
```

If we need the cross-account sentence, we need to rework what's here, but I don't understand why it's in this var.

```suggestion
  description = "The type of endpoint to use for communicating with the Key Protect or Hyper Protect Crypto Services instance. Possible values: `public`, `private`. Applies only if `existing_cos_kms_key_crn` is not specified."
```